source/arriving/setup_computer.rst

```diff
@@ -157,12 +157,43 @@ Nextflow
 
 TODO
 
-Super computers
-"""""""""""""""
+High performance computers
+""""""""""""""""""""""""""
 
-TODO
+The first use of a computing platform can be tricky but you'll get used to it. Here we explain how to get started on Beluga, one of the high performance computers (HPC) of the Digital Research Alliance of Canada (formerly Compute Canada). 
+
+Connect to Beluga via ssh with :bash:`ssh USER@beluga.computecanada.ca`.
+
+On your first visit, you will probably want to edit your .bashrc with your preferences. Since VSCode will not be available, you will have to use an editor built in the terminal like Nano (:bash:`nano ~/.bashrc`) or Vim (:bash:`vim ~/.bashrc`). Please refer to the :ref:`ref_linux` if you don't know these tools.
+
+Everytime you log in Beluga, you will need to load the modules necessary for your needs (scilpy, tractoflow, etc). Here are the modules currently needed for running Nextflow pipelines:
+
+.. code-block:: bash
+
+    module load StdEnv/2020
```

Comment:
Based on https://docs.alliancecan.ca/wiki/Standard_software_environments#StdEnv/2020 `StdEnv/2020` is still actively supported but not sure... @arnaudbore ?

Reply:
Yes it is!